frontend/src/lib/components/dialogs/FileSelectModal.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+	const { files, open, onSelect, onClose } = $props<{
```

Comment:
Please be consistent and create a FileSelectModalProps interface like all other components we have so far